views/components/ConfigDropdown.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react'
+import Select from '@material-ui/core/Select'
+import MenuItem from '@material-ui/core/MenuItem'
+import FormControl from '@material-ui/core/FormControl'
+import InputLabel from '@material-ui/core/InputLabel'
+
+const ConfigDropDown = ({
+  configurations,
+  updatePreferences,
+  currentConfig,
+  classes,
+}) => {
+  const [selectValue, setSelectValue] = useState('')
```

Comment:
The object comes equipped with an _id {
    star: { CA: [], LA: [] },
    sort: 0,
    config: '62f3c9ec5f6c21773c3e9a29',
    complete: { CA: [] }
  }

Reply:
hmm so there is an object called `currentConfig` and an array of objects called `configurations` but the objects are different?

Reply:
The currentConfig object is stored as preferences in the user collection and the configurations are the configurations owned by the user that live in the config collection

Reply:
ok I think this would be easier to grok if we change the name of `currentConfig` to `currentPreferences` or something similar. I still think we could remove the useEffect and state management from this component by using the config ID to find the current configuration from the configurations array. Does that make sense?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/129/commits/75fa4e051ac01363f4567f566d2f67ddf2280f34